appinventor/components/src/com/google/appinventor/components/runtime/TextBoxBase.java

```diff
@@ -499,6 +539,30 @@ public void TextColor(int argb) {
     }
   }
 
+  @SimpleFunction(description = "Repositions the cursor of the TextBox before the character at the given" +
+   " 1-indexed position. If the given position is larger than the length of the `TextBox`, the cursor will be moved" +
+   " to the end of the text; and if the given position is smaller or equal to 1, the cursor will be moved to the start.")
```

Comment:
Code style would have the `+` operator at the start of the following line rather than trailing at the end.

Reply:
Updated.